statsmodels/graphics/tests/test_regressionplots.py

```diff
@@ -111,6 +111,18 @@ def setup(self):
         y = Series(y, name="outcome")
         res = sm.OLS(y, exog0).fit()
         self.res = res
+        data = DataFrame(exog0, columns=["const", "var1", "var2"])
+        data['y'] = y
+        self.data = data
+
+class TestPlotFormula(TestPlotPandas):
+    def test_one_column_exog(self):
```

Comment:
I guess this needs to be decorated with a `skipif` when matplotlib is not available. 
Can you check the other test functions to see what we do here?